src/platforms/flutter/integrations/app-start-instrumentation.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: App Start Instrumentation
+description: "Learn more about the Sentry App Start Instrumentation for the Flutter SDK."
+caseStyle: camelCase
+supportLevel: production
+sdk: sentry.dart.flutter
+categories:
+  - mobile
+---
+
+Sentry's App Start Instrumentation provides insight into how long your application takes to launch.
+
+<Note>
+
+The App Start Instrumentation is shipped with Sentry's Flutter SDK automatically.
+
+</Note>
```

Comment:
Do we need this note? We don't have one on the Slow and Frozen Frames Instrumentation page, for example.

```suggestion
```